lib/ssl/src/tls_handshake_1_3.erl

```diff
@@ -922,6 +941,16 @@ do_wait_ee(#encrypted_extensions{extensions = Extensions}, State0) ->
     {Ref, Maybe} = maybe(),
 
     try
+        %% RFC 6066: handle received/expected maximum fragment length
```

Comment:
It would be cleaner if this block of code would be factored into a separate function following the pattern e.g.
```erlang
Maybe(handle_max_fragment_length(State0, Extensions)),
```